pycds/alembic/versions/5c841d2c01d1_reject_whitespace_on_meta_vars.py

```diff
@@ -0,0 +1,78 @@
+"""Updates text columns in meta_var to strip newlines and add constraints
+preventing further whitespace from being added.
```

Comment:
This is unusually long for a migration description. However, others in our stable are short and unhelpful. Can this be reduced just a little? And maybe on one line?

Reply:
I didn't realise these appear in the `alembic history` command. I've reduced it to "Add whitespace constraints to meta_vars"